project.mml

```diff
@@ -1316,7 +1353,23 @@ Layer:
             way,
             place,
             name
-          FROM planet_osm_point
+          FROM
+          (SELECT
+              way,
+              place,
+              name,
+              tags,
+              NULL AS way_area
+            FROM planet_osm_point
+          UNION ALL
+          SELECT
+              way,
+              place,
+              name,
+              tags,
+              NULL AS way_area
+            FROM planet_osm_polygon
```

Comment:
```
from planet_osm_polygon a
where not exists (
    select 
    from planet_osm_point p 
    where a.place=b.place 
        and a.name=b.name 
        and ST_Intesects(a.geom, b.geom)
)
```

Reply:
Nice idea.

One thing I noted. Should:

**from planet_osm_point p**

not be:

**from planet_osm_point b**

?